requirements/base.txt

```diff
@@ -40,7 +40,7 @@ boto==2.49.0
 google-api-python-client
 django-environ==0.4.5
 ipfshttpclient
-eth-utils==1.4.1
```

Comment:
what version of web3py are you using?

```
  File "app/grants/grants_transaction_validator.py", line 13, in <module>
    from web3.exceptions import (
ImportError: cannot import name 'TransactionNotFound'
```

I still cannot run this on `web3==4.5.0` which is whats currently in requrements/base.txt

Reply:
web3==5.7.0

Reply:
if youre going to upgrade the package version you need to update requirements/base.txt

and you'll need to make sure all the other web3py integrations dont work. i think its probably easier to just use the existing version..

Reply:
yesterday I upgraded, take a look at this pull.
https://github.com/gitcoinco/web/pull/6366

Reply:
ok so this PR depends on that one? maybe worth noting that in the description.

we'll also need good regression testing on that PR to make sure that nothing has broken across the parts of the site that rely on web3py.  that seems like a big increase in scope, which is why i was advising us to just stick with the existing version of web3py

Reply:
I understand, it is not a dependency but it is a plus for future issuses with interactions in the contract, I will leave this pull in the old version.

Reply:
can you downgrade your docker container to web3py 4.5 and try to run the code on this PR?  when i do that it gives me

```
  File "app/grants/grants_transaction_validator.py", line 13, in <module>
    from web3.exceptions import (
ImportError: cannot import name 'TransactionNotFound'

```

Reply:
@owocki add exception TransactionNotFound on c57959d60..fd469735a

Reply:
```python
class TransactionNotFound(Exception):
    """
    Raised when a tx hash used to lookup a tx in a jsonrpc call cannot be found.
    """
    pass        

def getTransaction___(txid):
    result = web3.eth.getTransaction(txid)
    if result is None:
        raise TransactionNotFound
    return result
    
def getTransactionReceipt___(txid):
    result = web3.eth.getTransactionReceipt(txid)
    if result is None:
        raise TransactionNotFound
    return result
```

To make it work on 4.5.0 we need it to act like it does on 5.7.0. Add these functions and replace the getTransaction and getTransactionReceipt references in the code with the new functions.

Reply:
@owocki what happened with this?

Reply:
i dont understand the question?